sgp4/export.py

```diff
@@ -0,0 +1,101 @@
+"""
+Export orbit data to a Two-Line-Element representation.
+"""
+from sgp4.io import compute_checksum
+
+from sgp4.model import Satrec
+
+
+def export_tle(satrec):
+    """
+     Generate the TLE for the data in the `Satrec` object.
+
+    Forms the two string lines of the TLE and returns them as a Tuple.
+
+    Parameters
+    ----------
+    satrec : Satrec
+        Object that holds the orbit information
+
+    Returns
+    -------
+    (line1, line2) : (str, str)
+    """
+
+    # Define constants
+    from math import pi
+    deg2rad = pi / 180.0  # 0.0174532925199433
```

Comment:
Instead of computing this every time the function is called, try moving it outside the function as a constant `_deg2rad` out at module level. Try the same thing with the constant on the next line, too.